apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -9,18 +9,14 @@
 {# FP-1828: Move to Core, Remove from Here #}
 {% include 'snippets/fp-1828-css-for-core.html' %}
 
-<style>
-  .page-text {
-    margin-top: 27px;
-    margin-bottom: 27px;
-  }
-</style>
```

Comment:
**Reminder to Self.**
@wesleyboar, you removed this because using standard template and/or breadcrumbs vertically aligned page content correctly, and this style was an attempt to vertically align content without standard and/or breadcrumbs.